changelog.d/0-release-notes/pr-2149

```diff
@@ -0,0 +1 @@
+Internal endpoint for re-authentication (`GET "/i/users/:uid/reauthenticate"`) in brig has changed in a backwards compatible way. Spar depends on this change for creating a SCIM token with 2nd password challenge. It's best to deploy Brig before Spar.
```

Comment:
```suggestion
Deploy Brig before Spar.
```

The rest of the text CAN be put into the release notes, but should be in "internal changes".